packages/web-components/fast-foundation/src/accordion-item/stories/accordion-item.register.ts

```diff
@@ -122,12 +121,8 @@ const styles = () => css`
     }
 `;
 
-DesignSystem.getOrCreate(document.body)
-    .withPrefix("fast")
-    .register(
-        AccordionItem.compose({
-            baseName: "accordion-item",
-            template,
-            styles,
-        })()
-    );
+FoundationAccordionItem.define(FoundationAccordionItem, {
+    name: "fast-accordion-item",
+    template: createAccordionItemTemplate(),
+    styles,
+});
```

Comment:
After the full refactor is complete, the above code will become less verbose and should look like this:

```ts
FoundationAccordionItem.define({
    name: "fast-accordion-item",
    template: createAccordionItemTemplate(),
    styles,
});
```